package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -769,7 +768,7 @@ class WaterBridgeAnalysis(AnalysisBase):
     #: N, O, P, and S. Any other heavy atoms are assumed to have hydrogens
     #: covalently bound at a maximum distance of 1.5 Å.
     r_cov = defaultdict(lambda: 1.5,  # default value
-                        N=1.31, O=1.31, P=1.58, S=1.55)
+                        N=1.31, O=1.31, P=1.58, S=1.55) # noqa: E741
```

Comment:
```suggestion
                        N=1.31, O=1.31, P=1.58, S=1.55)  # noqa: E741
```
With the linters working again, pep8 issues like this are a lot easier to work out!